src/components/CollapsedList/CollapsedList.js

```diff
@@ -36,7 +38,7 @@ function CollapsedList() {
         onToggle={() => toggleActive(0)}
       >
         <div className="collapsed-content Proxima_Nova  ">
-          <P style={{ color: '#9C9C9C' }} size="m-400">
```

Comment:
As for me, in this file you should have some json file with text for every paragraoh, and map "Paragraphs" instead of hardcoding indexes in it
In this case, `initialState` should define as
` new Array(n).fill(false, 0, n)`, where `n` is length of imported array of paragraphs

Don't Repeat Yourself

@Motzart what do you think?

Reply:
good idea, agree

Reply:
i fix ir
thank!